Neurotrauma/Lua/Scripts/helperfunctions.lua

```diff
@@ -50,6 +78,9 @@ function NT.SurgicallyAmputateLimb(character,limbtype,strength,traumampstrength)
     limbtoaffliction[LimbType.LeftArm] = "tla_amputation"
     limbtoaffliction[LimbType.Head] = "th_amputation"
     HF.SetAffliction(character,limbtoaffliction[limbtype],traumampstrength)
+
+    NT.DislocateLimb(character,limbtype,-1000)
+    HF.SetAfflictionLimb(character,"gangrene",limbtype,0)
```

Comment:
humanupdate already clears gangrene on surgically amputated limbs, so I copied that over here so that it happens instantly upon removal to ensure swapping limbs still clears it.